package.json

```diff
@@ -2,40 +2,25 @@
   "name": "vrms-server",
   "version": "0.3.0",
   "description": "VRMS Server",
-  "main": "server.js",
   "scripts": {
-    "test": "mocha",
-    "start": "node server.js",
-    "dev": "nodemon server.js",
-    "client": "npm run start --prefix client",
-    "dev-start": "concurrently \"npm run dev\" \"npm run client\"",
-    "heroku-postbuild": "cd client && npm install && npm run build"
-  },
-  "author": "sarL3y",
-  "license": "ISC",
-  "devDependencies": {
-    "chai": "^4.2.0",
-    "concurrently": "^5.1.0",
-    "debug": "^4.1.1",
-    "mocha": "^6.2.3",
-    "supertest": "^4.0.2"
+    "start": "concurrently \"cd backend && npm run start\" \"cd client && npm run start\"",
+    "dev:api": "cd backend && npm run dev",
+    "dev:client": "cd client && npm run start",
+    "dev": "dotenv -e backend/.env -e client/.env concurrently \"npm run dev:api\" \"npm run dev:client\"",
+    "cy:run": "dotenv -e client/.env -- cross-var cross-env CYPRESS_baseUrl=%CLIENT_URL% cypress run",
+    "test:cy": "dotenv -e ./backend/.env -e ./client/.env cross-var start-test dev:api %BACKEND_PORT% dev:client %CLIENT_PORT% cy:run",
+    "test:backend": "cd backend && npm run test",
+    "test:client": "cd client && npm run test",
+    "test:all": "NODE_ENV=test npm run test:client && npm run test:backend && npm run test:cy"
```

Comment:
My thought is that we should be able to set our scripts in each directories' package.json and then call to those scripts.

This takes a lot of care and attention to create. Thank you 🙇 

(see above package.json comments from client if you need more info)